beets/library.py

```diff
@@ -451,85 +451,85 @@ class Item(LibModel):
     _table = 'items'
     _flex_table = 'item_attributes'
     _fields = {
-        'id':       types.PRIMARY_ID,
-        'path':     PathType(),
+        'id': types.PRIMARY_ID,
+        'path': PathType(),
         'album_id': types.FOREIGN_ID,
 
-        'title':                types.STRING,
```

Comment:
It's a little sad to lose this table-like formatting, which did make this listing more readable, but I suppose it's not that big of a deal.

Reply:
Happy to reformat